test/application_system_test_case.rb

```diff
@@ -62,4 +62,42 @@ def user(user_or_fixture)
       users(user_or_fixture)
     end
   end
+
+  # In the post form, this method will select the given tag.
+  #
+  # @param tag_name [String] the name of the tag
+  # @param create_new [Boolean] whether creating a new tag is allowed (default false)
+  def post_form_select_tag(tag_name, create_new = false)
+    # First enter the tag name into the select2 search field for the tag
+    within find_field('Tags (at least one):').find(:xpath, '..') do
+      find('.select2-search__field').fill_in(with: tag_name)
+    end
+
+    # Get the first item listed that is not the "Searching..." item
+    first_option = find('#select2-post_tags_cache-results li:first-child') { |el| el.text != 'Searching…' }
```

Comment:
Careful: the ellipsis after "Searching" is a single Unicode character here, but I believe it's three dots in the actual UI - worth checking this actually picks the right tag.

Reply:
I actually fought for quite a while with getting this to work correctly because I was using just 3 dots (and then the test fails). When getting the exact text of this element, it turned out that it was a single character.

That said, i will add an assertion that verify that the right tag is selected

Reply:
Huh. I thought this was something we set - having just looked, the "Searching" text actually comes from select2, so that makes more sense.